packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -543,7 +549,7 @@ test.serial('STORY01: advancing happy path for 100 USDC', async t => {
     bridges: { snapshot, since },
     mint,
   } = t.context;
-  const cust1 = makeCustomer('Carl', cctp, txPub.publisher, feeConfig);
+  const cust1 = makeCustomer('Carl', cctp, txPub.publisher, feeConfig, storage);
```

Comment:
```suggestion
  const cust1 = makeCustomer('Carl', cctp, txPub.publisher, feeConfig);
```
all the methods of `cust1` already have access to `context` and thus `storage`.

`t` is passed in with every call. That could be refactored but is out of scope